versioned_docs/version-2.x/magento1/installation.mdx

```diff
@@ -223,6 +223,12 @@ For more security add a random key on `frontcommerce_secret_key` in your
 
 <SinceVersion tag="2.27" />
 
+:::info
+
+This feature requires Front-Commerce Magento 1 module version 1.6.0 or higher.
```

Comment:
given https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2758#note_108477 this should also be mentioned in the 2.27 upgrade guide that the Magento1 module **must** be updated to at least 1.6.0

Reply:
Added a section in migration guide.